### PR TITLE
Documentation Directory and linting

### PR DESCRIPTION
**Pull request by Shtoyan**
- Moved `TODO` to Docs dir.
- Checkboxes for `TODO`.
- Created a proper `CHANGELOG` (needs to be filled later).
- Fixed few tabs -> x4 spaces.
- Linted all md files to be more fancy and readable.
- Added crosslinks between md files and `README`.

### CODE REVIEW

**Review by theengineertcr — COMMENTED** [review]
Thanks for the help, although for some reason I didn't receive any notification about these commits yesterday and just now saw them. 

The latest commit had GunnerGLProjectile go through multiple changes so I won't be able to accept the pull request for that file specifically, although I definitely want to accept the other commits, but I'm not sure if that's possible. I'm currently reading/watching some videos right now to learn about this. 

Cheers!

**Comment by theengineertcr** [comment]
Seems like that pulling specific commits to a remote repository isn't possible, so I'll just do a review instead.

File: KFOldSchoolZedsMod/classes/GunnerGLProjectile.uc [hunk 1]
```diff
@@ -48,8 +48,8 @@ simulated function ProcessTouch(Actor Other, Vector HitLocation)
 
     if( !bKillMomentum )
     {
-	   Explode(HitLocation,Normal(HitLocation-Other.Location));
-	}
+       Explode(HitLocation,Normal(HitLocation-Other.Location));
+    }
 }
 
 defaultproperties
```
Comment (theengineertcr) [inline KFOldSchoolZedsMod/classes/GunnerGLProjectile.uc]: Obsolete, latest commit has proper formatting.

File: README.md [hunk 2]
```diff
@@ -14,13 +14,13 @@ Select the Killing Floor 2.5 Gametype and start a game.
...
+    ZombieClot_OS
+    ZombieBloat_OS
+    ZombieGorefast_OS
+    ZombieCrawler_OS
+    ZombieStalker_OS
+    ZombieSiren_OS
+    ZombieRangedPound_OS
+    ZombieScrake_OS
+    ZombieFleshpound_OS
+    ZombieBoss_OS
```
Comment (theengineertcr) [inline README.md]: This is fine.

File: Docs/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Changelog
+
+> Return back to [README](../README.md#documentation)
+
+## [2022.05.14]
+
+- Fill me later!
```
Comment (theengineertcr) [inline Docs/CHANGELOG.md:7]: This is a very good suggestion! Thanks a lot for this.

File: Docs/TODO.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
+# To-Do
```
Comment (theengineertcr) [inline Docs/TODO.md:1]: This is good as well.

File: README.md [hunk 5]
```diff
@@ -8,19 +8,29 @@ Copy all of the contents within the zip file into your Killing Floor directory.
 
 ## Usage
 
-Select the Killing Floor 2.5 Gametype and start a game.
+Select the Killing Floor 2.5 Gametype and start a game. Or for a dedicated server:
```
Comment (theengineertcr) [inline README.md]: Need to expand on the dedicated server portion afterwards. The additional formatting is nice.

File: Docs/TODO.md [hunk 6]
```diff
@@ -4,37 +4,38 @@
 
 ## Gameplay Tweaks
 
-- Make Fleshpound Avoid Area's only spawn when enraged and make Avoid Area extend a couple of meters infront of Fleshpounds to make zeds "Clear the way"
+- [ ] Make Fleshpound Avoid Area's only spawn when enraged and make Avoid Area extend a couple of meters infront of Fleshpounds to make zeds "Clear the way"
   - Currently, Fleshpound Avoid Area has been temporarily removed until this is done.
-- Fleshpound/Ranged Pound head hitboxes too generous
-- Bloat acid stays for far too long, make it go away earlier.
+- [ ] Fleshpound / Ranged Pound head hitboxes too generous
+- [ ] Bloat acid stays for far too long, make it go away earlier.
```
Comment (theengineertcr) [inline Docs/TODO.md]: This file has been updated, but I'll hold off on pushing any further commits on this until I can pull the directory changes.

**Review by theengineertcr — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (Shtoyan) [reply README.md]: I reformatted the message a bit, but believe me server admins know what this means and where to use that line :d

**Review by Shtoyan — COMMENTED** [review]
Notifications doesn't work for me as well *shrugs* so we can blame them for this PR's bad timing. I'll ask you in PM for further updates / PR's so we can synchronize.

**Pull request merged**